spec/lib/es/client_spec.rb

```diff
@@ -125,6 +125,7 @@
         config[:elasticsearch][:compression] = false
         result = subject.connection_config(config[:elasticsearch], '0.0.0-test')
         expect(result[:compression]).to be false
+        expect(system_logger).to have_received(:info).with('ES connection compression is enabled')
         expect(system_logger).to have_received(:info).with('ES connection compression is disabled')
```

Comment:
Same situation here; should it not be only the second log?

Reply:
Will address and will make sure you don't need to leave comments like these on the next PR 🚀